railties/lib/rails/engine/configuration.rb

```diff
@@ -88,9 +87,11 @@ def autoload_paths
       end
 
       def webpacker_path
-        if File.file?("#{Rails.root}/config/webpacker.yml")
-          YAML.load_file("#{Rails.root}/config/webpacker.yml")[Rails.env]["source_path"]&.gsub("app/", "")
-        else
+        return "javascript" unless File.file?("#{Rails.root}/config/webpacker.yml")
```

Comment:
Can you move this check under the `begin` for cache result?

Reply:
do we need to cache result here, is it called often enough to be cached?

Reply:
its called a few times during boot.  worth it even if its a slight improvement